src/app/integrations/list/list.component.html

```diff
@@ -35,161 +37,212 @@ <h4 class="modal-title pull-left">Confirm {{ getActionTitle() }}</h4>
+            <!-- Description -->
+            <div class="col-xs-8 description"
+                 (click)="goto(integration)">
+              <div class="list-group-item-text">
+                {{ integration.description }}
+              </div>
+            </div>
+          </div>
+
+          <!-- Group 3: Metrics & Actions -->
+
+          <div class="col-xs-4 group metrics-actions">
+
+            <!-- Active/Inactive Labels -->
+
+            <div class="col-xs-5 active-inactive">
+              <div class="list-view-pf-additional-info-item">
+
+                <!-- In Progress -->
+                <!--
+                <div *ngIf="(integration.currentStatus !== integration.desiredStatus) && !integration.statusMessage">
+                  <div class="spinner spinner-sm spinner-inline"></div>
+                </div>
+                -->
+                <div class="in-progress">
+                  <h3>
+                    <div class="spinner spinner-sm spinner-inline"></div>
+                    In Progress
+                  </h3>
+                </div>
+
+                <div *ngIf="integration.currentStatus === integration.desiredStatus">
+                    <span *ngIf="integration.currentStatus === 'Activated'"
+                          class="label label-success"
+                          [tooltip]="statTemplate"
+                          (click)="requestDeactivate(integration)">Active</span>
+                  <span *ngIf="integration.currentStatus === 'Deactivated'"
+                        class="label label-default"
+                        [tooltip]="statTemplate"
+                        (click)="requestActivate(integration)">Inactive</span>
+                  <span *ngIf="integration.currentStatus === 'Draft'"
+                        class="label label-primary"
+                        [tooltip]="statTemplate"
+                        disabled>Draft</span>
+                  <span *ngIf="integration.currentStatus === 'Deleted'"
+                        class="label label-danger"
+                        [tooltip]="statTemplate"
+                        disabled>Deleted</span>
+                </div>
+              </div>
+            </div>
+
+
+            <!-- Metrics & Actions -->
+
+            <div class="col-xs-7">
+
+              <!-- Uses -->
+
+              <div class="col-xs-7 uses">
+                <p>{{ integration.timesUsed || 0 }} uses</p>
+              </div>
+
+
+              <!-- Pass/Fail -->
+
+              <div class="col-xs-4 pass-fail">
+
+                <!-- OK Status -->
+
+                <i *ngIf="!integration.statusMessage"
+                   class="pficon pficon-ok"></i>
+
+                <!-- Not OK Status -->
+
+                <template #statTemplate>{{integration.statusMessage}}</template>
+                <i *ngIf="integration.statusMessage"
+                   class="pficon pficon-error-circle-o"
+                   [tooltip]="statTemplate"
+                   placement="bottom"
```

Comment:
I like for it to be on top better, but one error was so huge it disappeared into the browser URL bar, so had to make the placement "bottom".